modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/KaleoFormsNewSubmissions.macro

```diff
@@ -78,6 +167,20 @@ definition {
 			locator1 = "KaleoFormsNewSubmissions#LIST_TABLE_NAME_STATUS");
 	}
 
+	macro viewStatusLabel {
+		AssertElementPresent(
+			key_status = "${lbstatus}",
+			locator1 = "KaleoFormsNewSubmissions#SUBMISSION_STATUS_LABEL");
+	}
+
+	macro viewVersionOnDetails {
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "View");
+
+		AssertElementPresent(
+			locator1 = "KaleoFormsAdmin#WORKFLOW_VERSION",
+			value1 = "1.0");
```

Comment:
Please, pass "value1" as a variable, cause the workflow version value can be changed. Thanks!